src/meep.hpp

```diff
@@ -1490,6 +1490,7 @@ class gaussianbeam {
   gaussianbeam(const vec &x0, const vec &kdir, double w0, double freq,
                double eps, double mu, std::complex<double> EO[3]);
   void get_fields(std::complex<double> *EH, const vec &x);
+  std::complex<double> get_E0(int n) { return E0[n]; };
```

Comment:
Both `get_fields` and `get_E0` should be `const` methods since they don't modify the object:
```
void get_fields(std::complex<double> *EH, const vec &x) const;
std::complex<double> get_E0(int n) const { return E0[n]; };
```

Reply:
Fixed.